datasets/coco2014/metadata.json

```diff
@@ -236,6 +236,7 @@
     {
       "name": "categories",
       "@type": "ml:RecordSet",
+      "ml:isEnumeration": "true",
```

Comment:
I think JSON supports `true` (without using a string). Should you use a boolean or a string value?

Reply:
Absolutely, thanks!